.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -4,30 +4,36 @@ import { random } from '../support/utils';
 it('allows creating a new user', () => {
   cy.login();
   cy.get('[href*="/users"]').click();
-  cy.get('#createUser').click();
+  cy.get('button[data-testid="add-user"]').click();
   cy.fixture('users').then((user) => {
-    cy.get('#username').type(`${user.newUser.username}.${random}`);
-    cy.get('#email').type(`${user.newUser.email}.${random}`);
-    cy.get('#firstName').type(`${user.newUser.firstName}.${random}`);
-    cy.get('#lastName').type(`${user.newUser.lastName}.${random}`);
+    cy.get('#kc-username').type(`${user.newUser.username}.${random}`);
+    cy.get('input[data-testid="firstName-input"]').type(`${user.newUser.firstName}.${random}`);
+    cy.get('input[data-testid="lastName-input"]').type(`${user.newUser.lastName}.${random}`);
   });
-  cy.contains('button', 'Save').click();
-  cy.contains('.alert', 'Success');
+  cy.get('button[data-testid="create-user"]').click();
+  cy.contains('h4', 'The user has been created');
```

Comment:
Maybe the selector is not necessary?

```suggestion
  cy.contains('The user has been created');
```

Reply:
I didn't know this. Thanks!